ci/kind/kind-setup.sh

```diff
@@ -244,25 +245,68 @@ function configure_extra_networks {
   done
 }
 
+function get_bridge_id {
+  local bridge_id=$(docker network inspect kind -f {{.ID}})
+  local return_value=$?
+  if [ $return_value -eq 0 ]; then
+        echo "$bridge_id"
+    else
+        echo "Unable to get kind bridge id, Docker command failed with exit code $return_value"
+        exit 1
+    fi
+}
+
+# add_kind_ipam_routes  add routes for non-ipam test-pods.
+function add_kind_ipam_routes {
+     kubectl get nodes -o wide --no-headers=true |  awk '{print $1}' | while read nodes; do
+      local pod_cidr=$(kubectl get nodes "${nodes}" -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+      local node_ip=$(kubectl get nodes "${nodes}" -o wide | awk 'FNR == 2 {print$6}')
+      sudo ip route add "$pod_cidr" via "$node_ip"
+     done
+}
+
+# delete_kind_ipam_routes delete the routes of non-ipam test-pods from the Kind testbed.
+function delete_kind_ipam_routes { 
+    cluster=$(kind get clusters)
+    if [ ! -z "$cluster" ]; then
```

Comment:
I don't understand the check, could you explain?

Reply:
in the kind builder before creating the new kind cluster , we run `"./ci/kind/kind-setup.sh destroy kind" `cmd to delete the previous run kind cluster , and if there is no cluster then "delete_kind_ipam_routes" function will return the error because of this cmd `kubectl get nodes -o jsonpath='{.items[*].spec.podCIDR}'`, that's why I am checking for the cluster first.